layers/meta-resin-allwinner/recipes-kernel/rtl8189fs/rtl8189fs/0001-Disable-debug-messages.patch

```diff
@@ -0,0 +1,25 @@
+From f2513e71dffdb207a7d71f7c52bac0f67b9fbcb4 Mon Sep 17 00:00:00 2001
```

Comment:
Can you change the commit log to show the component

> rtl8189fs: 

instead of using the patch name, please?

Reply:
done